### ISSUE TITLE
Multiple zookeeper

### ISSUE THREAD

**Issue by hieutrtr** [ches/docker-kafka#7]
Hi Ches,

Do you have any ideas about multiple zookeeper using docker. Could you suggest some solution to do that.

Thanks and Best Regards,
Hieu Tran Trung.

**Comment by xrl** [comment]
Yes, you can set `ZOOKEEPER_CONNECTION_STRING`. There's an example in the README, but essentially: 

```
docker run -e ZOOKEEPER_CONNECTION_STRING=zk-1:2181,zk-2:2181 $REST_OF_RUN_CMD
```

should get you going.
